l5kit/l5kit/dataset/agent.py

```diff
@@ -11,6 +11,10 @@
 from .ego import EgoDataset
 from .select_agents import TH_DISTANCE_AV, TH_EXTENT_RATIO, TH_YAW_DEGREE, select_agents
 
+# WARNING: changing these values impact the number of instances selected for both train and inference!
+MIN_FRAME_HISTORY = 10  # minimum number of frames an agents must have in the past to be picked
+MIN_FRAME_FUTURE = 0  # minimum number of frames an agents must have in the future to be picked
```

Comment:
MIN_FRAME_HISTORY and _FUTURE are +1 right, ie MIN_FRAME_HISTORY = 10 means 10 past frames + current right? 
Does it make sense to set _FUTURE to 0 then, or should it not be 1? (As agents with no future frame are never used.)

Reply:
only history is +1, so 10 means present plus 10 frames.
future 0 means the future is not considered (e.g. can all be invalid). I also think 1 is a better value, let's put 1 @oliver-scheel